### PR TITLE
arranged mentor page

### PR DESCRIPTION
**Pull request by rishuraj1**
## Related Issue

Closes: #514 

## Description of Changes

Changes:
- wrote fresh code keeping previous format
- utilized tailwind css to its fullest for better code readability.
- added: better styles and icons
- made the page fully responsive.

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [ ] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots

image

image

image

image

image

### CODE REVIEW

**Comment by anmode** [comment]
Just quick UI overview:
1. Instead blue Icon can we have our purple color/theme color 
2. FAQ are very big in size should be reduced..
3. Mobile optimization needed...
4. Please replace the vector(copyright issue), or change color to ours

have a look:https://mentro.tech/mentor/

**Comment by rishuraj1** [comment]
Hey @anmode, I have done the needed changes according to the above feedback,

- Forr the image/svg part, please have a look and tell me if this is what you were referring to. I have added a background of color purple to it.

- If the image is to be changed, can you please provide me with the image/vector to replace if you have already one.

- I have reduced the font sizes for the FAQ section with added responsiveness.
-  Made the page responsive for mobile devices.

image

image


dimensions: 410 x 722

image

image

image


Thank You !

**Comment by anmode** [comment]
Some fixes:
1. Color code is wrong... this is our color: #845ec2
2. Theme image or main vector image is not good please replace or you can create awesome animation in other PR. It will of level 3 too. ( I'll provide vector then if you are not up for animation ) no worries :)
3. actully i was also talking about color of icon as cohort courses from to our color.

btw it is looking nice just needs some optimization.. let me know I'll set gmeet too if need

**Comment by rishuraj1** [comment]
Alright @anmode , I get it now, will make necessary changes. Can you please provide me the new image. I couldn't find one. Thanks

**Comment by anmode** [comment]
Sorry @rishuraj1 i took time due to grab2.0 launch..
so here : https://www.freepik.com/free-vector/pensioners-financial-literacy-finance-education-savings-management-investment-awareness-consultant-explaining-finance-system-basics-elderly-people-vector-isolated-concept-metaphor-illustration_12083567.htm#query=mentorship&position=15&from_view=search&track=sph

**Comment by anmode** [comment]
@rishuraj1 Need UX improvents too.. based on the page theme ( a mentor page to show what they will get from us you take refer from https://mentro.tech/mentor/, actually we have just copied same UX toh may be issue hoskta h.


2nd When I hover those services...cyan is coming.. change it to other color from this our theme pallet: b1c5e364-a2ad-40bb-b3ba-4101a0b05016

**Comment by rishuraj1** [comment]
Hey @anmode made some additional changes according to above theme pallete, I have changed the bg from cyan to #FBEAFF and used #845EC2 wherever needed. also I added another section "Become a Mentor" and some text under it so the page looks filled. I plan to add another section "testimonial" inspired from the reffered website above but, will open seperate issue for that. For now, Do let me know if any additional changes are required. Thank You !

**Comment by anmode** [comment]
Wow looks cool, Hopefully this will be last fixes.. I'll explain in detail too... thanks for coperating
1. Remove the background of vector please
Screenshot 2023-07-26 at 8 26 17 PM

Screenshot 2023-07-26 at 8 27 28 PM

2.  In this section:
Screenshot 2023-07-26 at 8 27 48 PM

1. icon color before hover should be white
2. background color of rectangular box which is light pink should be replace by darker grey then background color.
3. On hover remove the cyan from icon instead you can have green from our pallet.

3. In this section:
Screenshot 2023-07-26 at 8 40 45 PM
some fixes as box should also close on clicking anywhere on screen too.

On open you see pink background color... same here replace by darker grey then background

**Comment by rishuraj1** [comment]
@anmode  , done with the changes. plz check

**Comment by anmode** [comment]
ok let's break this issue...
please next PR need the mobile optimization and some UX fixes as faq and testimonials....
thanks for making it!

File: public/assets/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export { default as MentorImg } from './img/MentorImg.png';
+export { default as BecomeMentor } from './img/BecomeMentor.png';
+export { default as GSoC } from './img/company/GSoC.svg';
+export { default as microsoft } from './img/company/microsoft.svg';
+export { default as mlh } from './img/company/mlh.svg';
+export { default as hackerRank } from './img/company/hackerRank.svg';
```
Comment (anmode) [inline public/assets/index.js:6]: liked it!

**Review by anmode — APPROVED** [review]
cool

**Pull request merged**